### PR TITLE
grammar-and-gender

### PR DESCRIPTION
**Pull request by gonyere**
gendered terms, and some minor grammar edits, mostly along with them, but a few others nearby as well

### CODE REVIEW

File: docs/check_source.asciidoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ cd openSUSE-release-tools
 ln -sr ./osc-check_source.py ./osclib ~/.osc-plugins
 -------------------------------------------------------------------------------
 
-The program will search also some external scripts (like
+The program will als search some external scripts (like
```
Comment (coolo) [inline docs/check_source.asciidoc:35]: this looks a missing o

File: docs/staging.asciidoc [hunk 2]
```diff
@@ -19,13 +19,13 @@ This plugin is used by the release manager to move requests into
 staging projects to evaluate the behavior and inter-compatibility of
 different changes working together.
 
-An staging project is a mere project linked to a subset of Factory
+An\ staging project is a mere project linked to a subset of Factory
 (MinimalX) , but with some packages that are different as a result of
-one or several requests.  When the release manager assign a request
-into a staging project, this request is review inside this project.
+one or several requests.  When the release manager assigns a request
+in a staging project, this request is review inside th project.
```
Comment (coolo) [inline docs/staging.asciidoc:25]: this looks a missing e

File: docs/staging.asciidoc [hunk 3]
```diff
@@ -19,13 +19,13 @@ This plugin is used by the release manager to move requests into
 staging projects to evaluate the behavior and inter-compatibility of
 different changes working together.
 
-An staging project is a mere project linked to a subset of Factory
+An\ staging project is a mere project linked to a subset of Factory
```
Comment (coolo) [inline docs/staging.asciidoc:22]: Has this backslash a significance?

File: docs/testing.asciidoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ nosetests
 Structure of the suite
 ----------------------
 
-Each object is containing functions for the individual tests so split them per
+Each object is contains functions for the individual tests so split them per
```
Comment (coolo) [inline docs/testing.asciidoc:28]: the is looks too much now

**Review by coolo — CHANGES_REQUESTED** [review]
Thanks for the feedback. I found some typos though.

And I'm happy that you didn't find any assumed male roles :)

One more thing: Please use a little more descriptive commit log - what you wrote as PR summary would do.

**Comment by gonyere** [comment]
Hi, thanks for the comments! I'm kinda new at this, I'll make sure my commit log is more descriptive next time!

**Comment by coolo** [comment]
Can you please fix the typo in check_source? Then I can merge this

**Review by gonyere — COMMENTED** [review]
fixed also

**Pull request merged**